single-sign-on/manage/index.md

```diff
@@ -18,6 +18,30 @@ To add a guest to your organization in Docker Hub if they aren’t verified thro
 2. Select **Add Member**, enter the email address, and select a team from the drop-down list.
 3. Select **Add** to confirm.
 
+## Audit your domain for users
+
+When your organization has configured SSO and registered a domain, you can audit your domain. Auditing your domain will identify uncaptured users that have an email associated with your domain, but they aren't a member of your organization.
```

Comment:
@craig-osterhout do we want to also set as requirement that beyond having the domain register it's need to be verified?

Reply:
propose "...has configured SSO and verified a domain..."

Also - this might change in future when we decouple domain verification e.g. "When your organization has verified a domain" (not needed now, just FYI)

Reply:
propose "....identify uncaptured users that have authenticated with an email associated with your verified domain..."